resource_management_mobile/src/app/shared/components/resource-card/resource-card.component.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../../../theme/theme.scss";
```

Comment:
Don't import theme.scss every where declare it in angular.json or use as common global.scss

Reply:
Done